martian-derive/tests/mro/test_main_only.mro

```diff
@@ -9,6 +9,7 @@ stage SUM_SQUARES(
     out float   sum_sq,
     src comp    "adapter martian sum_squares",
 ) using (
-    mem_gb  = 4,
-    threads = 2,
+    mem_gb   = 4,
+    threads  = 2,
+    disabled = UPSTREAM_STAGE.disabled,
```

Comment:
This would fail `mro check`.  `disabled` is not a legal modifier in the `using` clause of a stage definition.

Reply:
Also, `UPSTREAM_STAGE` would be undefined here.